src/view/utils.py

```diff
@@ -4,8 +4,33 @@
 from src.view import ui
 
 
-def is_leaf(dic, d):
-    values = dic.get(d)
+def is_leaf(dico, key):
+    """
+    This function check if a key is the penultimate in a dictionnary depth
+
+    Example:
+    with a dictionnary with this structure:
```

Comment:
Just to mention, in english there is only one 'n' in dictionary.

Reply:
done